script/hassfest/services.py

```diff
@@ -65,25 +65,23 @@ def grep_dir(path: pathlib.Path, glob_pattern: str, search_pattern: str) -> bool
 
 def validate_services(integration: Integration):
     """Validate services."""
-    # Find if integration uses services
-    has_services = grep_dir(
-        integration.path,
-        "**/*.py",
-        r"(hass\.services\.(register|async_register))|async_register_entity_service|async_register_admin_service",
-    )
-
-    if not has_services:
-        return
-
     try:
         data = load_yaml(str(integration.path / "services.yaml"))
```

Comment:
Probably would be slightly faster to stat it to see if it exists but probably not worth it as this will be much faster